lib/embedded_tweet_view.dart

```diff
@@ -182,25 +187,25 @@ class EmbeddedTweetView extends StatelessWidget {
                   ),
                   (_tweetVM.quotedTweet != null)
                       ? Padding(
-                          padding: EdgeInsets.only(top: 8.0, bottom: 10),
-                          child: QuoteTweetViewEmbedded(
-                            _tweetVM.quotedTweet!,
-                            textStyle: TextStyle(
-                                color:
-                                    (darkMode) ? Colors.white : Colors.black),
-                            clickableTextStyle: defaultQuoteClickableTextStyle,
-                            userNameStyle: (darkMode)
-                                ? defaultEmbeddedDarkQuoteUserNameStyle
-                                : defaultQuoteUserNameStyle,
-                            userScreenNameStyle:
-                                defaultQuoteUserScreenNameStyle,
-                            backgroundColor: null,
-                            borderColor: null,
-                            onTapImage: onTapImage,
-                            autoPlayVideo: autoPlayVideo,
-                            enableVideoFullscreen: enableVideoFullscreen,
-                          ),
-                        )
+                    padding: EdgeInsets.only(top: 8.0, bottom: 10),
```

Comment:
Looks like you reformatted te code - could you bring it back?

Reply:
ok, i will do it right now. thanks for your comment